lua/wikis/hub/MainPageLayout/data.lua

```diff
@@ -0,0 +1,180 @@
+---
+-- @Liquipedia
+-- page=Module:MainPageLayout/data
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Lua = require('Module:Lua')
+
+local Link = Lua.import('Module:Widget/Basic/Link')
+local HtmlWidgets = Lua.import('Module:Widget/Html/All')
+local Li = HtmlWidgets.Li
+local Ul = HtmlWidgets.Ul
+
+local CONTENT = {
+	supporthub1 = {
+		heading = 'Want to Contribute?',
+		body = Ul{children = {
+			Li{
+				children = HtmlWidgets.B{children = Link{
+					link = 'Support/Getting started',
+					children = 'Getting started'
+				}}
+			},
+			Li{children = Link{
+				link = 'https://tl.net/mytlnet/register',
+				children = 'Create an account',
+				linktype = 'external',
+			}},
+			Li{children = Link{
+				link = 'Support',
+			}},
+			Li{children = Link{
+				link = 'lpcommons:Special:NewFiles',
+				children = 'Latest uploads',
+			}},
+		}},
+		padding = true,
+	},
+	supporthub2 = {
+		heading = 'Liquipedia',
+		body = HtmlWidgets.Fragment{children = {
+			Ul{children = {
+				Li{children = Link{
+					link = 'Support/Liquipedia',
+					children = 'Liquipedia',
+				}},
+				Li{children = Link{
+					link = 'Liquipedia:Alpha Wikis Program',
+					children = 'Starting a new wiki',
+				}},
+				Li{children = Link{
+					link = 'Liquipedia:Policy',
+					children = 'Policies',
+				}},
+			}},
+			HtmlWidgets.Br{classes = {'mobile-hide'}},
+		}},
+		padding = true,
+	},
+	supporthub3 = {
+		heading = 'LP Dev',
+		body = HtmlWidgets.Fragment{children = {
+			Ul{children = {
+				Li{children = Link{
+					link = 'Liquipedia:Changelogs',
+					children = 'Dev updates',
+				}},
+			}},
+			HtmlWidgets.Br{classes = {'mobile-hide'}},
+			HtmlWidgets.Br{classes = {'mobile-hide'}},
+			HtmlWidgets.Br{classes = {'mobile-hide'}},
+		}},
+		padding = true,
+	},
+	supporthub4 = {
```

Comment:
Is this the best variable names we could come up with? Maybe make the contact box called `contact` etc